web-flasher/offline/start-linux-mac.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# FIAT-HELL Web Flasher - offline spustenie (Linux / macOS)
+# Web Serial vyzaduje https alebo localhost, preto sa stranka servuje lokalne.
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)